docs/sources/auth/saml.md

```diff
@@ -0,0 +1,136 @@
+
+# Grafana SAML Configuration
+
+An example working configuration example looks like:
+
+```bash
+[auth.saml]
+enabled = true
+certificate_path = "/path/to/certificate.cert"
+private_key_path = "/path/to/private_key.pem"
+metadata_path = "/my/metadata.xml"
+max_issue_delay = 90s
+metadata_valid_duration = 48h
+```
+
+And here is a comprehensive list of the options:
+
+| Setting                   | Required | Description                                                                    | Default |
+|---------------------------|----------|--------------------------------------------------------------------------------|---------|
+| `eanbled`                 | No       | Whenever SAML authentication is allowed                                        | `false` |
+| `certificate|_path`       | Yes      | Base64-encoded string or Path for the SP X.509 certificate                     |         |
+| `private_key|_path`       | Yes      | Base64-encoded string or Path for the SP private key                           |         |
+| `idp_metadata|_path`      | Yes      | Base64-encoded string or Path for the IdP SAML metadata XML                    |         |
+| `max_issue_delay`         | No       | Duration, since the IdP issued a response and the SP is allowed to process it. | `90s`   |
+| `metadata_valid_duration` | No       | Duration, for how long the SP's metadata should be valid.                      | `48h`   |
+
+## `certificate` and `private_key`
+
+The SAML SSO standard uses asymmetric encryption to exchange information between the SP (Grafana) and the IdP. To perform such encryption, we need a public part and a private part. In this case, the X.509 certificate provides the public part, while the private key provides the private part.
+
+Grafana supports two ways of specifying both the `certificate` and `private_key`. Without a suffix (e.g. `certificate=`), the configuration assumes you've supplied the base64-encoded file contents. However, if specified with the `_path` suffix (e.g. `certificate_path=`) we treat it as a file path and attempt to read the file from the file system.
+
+## `idp_metadata`
+Expanding on the above, we'll also need the public part from our IdP for message verification. The SAML IdP metadata XML tells us where and how we should exchange the user information.
+
+Currently, we support two ways of specifying the IdP metadata. Without a suffix `idp_metadata=` we'll assume base64-encoded XML file contents, and with the `_path` suffix we treat it as a file path and attempt to read the file from the file system.
+
+## `max_issue_delay`
+
+To prevent SAML response replay attacks and internal clock skews between the SP (Grafana) and the IdP. We can set a maximum amount of time between the IdP issuing a response and the SP (Grafana) processing it.
+
+The configuration options is specified as a duration e.g. `max_issue_delay= 90s` or `max_issue_delay= 1h`
```

Comment:
Did you meant to put a space after `=` here?

Reply:
Great catch! 👌